sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -155,6 +166,9 @@ struct pi_command_list_info_t {
   ze_command_queue_handle_t ZeQueue{nullptr};
   // Keeps the ordinal of the ZeQueue queue group. Invalid if ZeQueue==nullptr
   uint32_t ZeQueueGroupOrdinal{0};
+  // Keeps the queue descriptor of the command list.
+  pi_command_list_and_desc_t ZeQueueDesc;
```

Comment:
I suggest using `ze_command_queue_desc_t` and remove the `ZeQueueGroupOrdinal` above

Reply:
it also seems redundant that `pi_command_list_info_t` holds `ze_command_list_handle_t` via `pi_command_list_and_desc_t`. can't this just be `ze_command_queue_desc_t` here?

Reply:
I've removed the ordinal and put it in the ZeQueueDesc. As explained above, we can't recover any info from the ze_command_queue_desc_t and so must remember all the fields used when creating it.

Reply:
Yes, it is redundant that CommandListMap is keyed with the command list and the pi_command_list_and_desc_t also contains the command list.
However, when placing things in the ZeComputeCommandListCache and ZeCopyCommandListCache maps, there is no command list key being used, and the command list can only be recovered from the entry type, pi_command_list_and_desc_t .

Reply:
Ok, I have fixed that redundancy. pi_command_list_and_desc_t is now named pi_command_list_desc_t, it does not contain the command list, and the entries in Ze{Compute/Copy}CommandListCaches are a pair of command list and pi_command_list_desc_t.